ext/node/ops/require.rs

```diff
@@ -445,33 +445,18 @@ pub fn op_require_path_basename(
 
 #[op2(stack_trace)]
 #[string]
-pub fn op_require_try_self_parent_path<
+pub fn op_require_maybe_cwd<
```

Comment:
Is this even needed? We already have ops that return cwd. Also, if there's no perm check we don't need `stack_trace` attr on `op2` and don't need to accept `P`

Reply:
Updated to use op_fs_cwd